nameserver/status.go

```diff
@@ -32,12 +33,15 @@ func NewStatus(ns *Nameserver, dnsServer *DNSServer) *Status {
 			entry.ContainerID,
 			entry.Addr.String(),
 			entry.Version,
-			entry.Tombstone})
+			entry.Tombstone,
+		})
 	}
 
 	return &Status{
 		dnsServer.domain,
+		dnsServer.upstream.Servers,
 		dnsServer.address,
 		dnsServer.ttl,
-		entryStatusSlice}
+		entryStatusSlice,
+	}
```

Comment:
I did it. I remember reading about never ending lines with `}` in Go because it makes `diffs` worse...